ssl4rs/data/transforms/batch.py

```diff
@@ -290,7 +290,6 @@ def get_batch_index(
     assert isinstance(batch_index, typing.Hashable), f"invalid batch index type: {type(batch_index)}"
     return batch_index
 
-
```

Comment:
this module seems to contain changes that should not happen or do not need to happen; these should be eliminated